src/recipe.js

```diff
@@ -7,12 +7,22 @@ class Recipe {
     this.image = recipe.image;
     this.tags = recipe.tags;
     this.ingredients = recipe.ingredients;
+    this.instructions = recipe.instructions
   }
-  calculateIngredientsCost() {
-    return this.ingredients.map(i => {
-      ingredientData.find(ingredient => ingredient === i);
-    });
+
+  calculateIngredientsCost(ingredientData) {
```

Comment:
If `this.ingredients` is an empty array does this return 0? Probably something that we can hardcode to handle as an exception pretty easily. Maybe we can add a sad-path test in the future, also.